app/panel/components/BuildingBlocks/CliqzFeature.jsx

```diff
@@ -51,6 +51,11 @@ class CliqzFeature extends React.Component {
 	}
 
 	_getStatus(active) {
+		const { current_theme, trackerCount } = this.props;
+		if (current_theme === 'palm-theme'
+			|| current_theme === 'leaf-theme') {
+			return active ? (<span>{trackerCount}</span>) : (<div className="rectangle" />);
+		}
 		return active ? t('on') : t('off');
 	}
```

Comment:
Why are we showing different types of information above the icons depending on theme? If this is an intended "extra info for plus members" type of feature (if so is it documented anywhere?), how come we are not also showing the numbers for the Dark Blue theme?

....I checked that the ticket and I see that the theme images have counts. Have we confirmed that this is as intended and isn't a remnant from when we used numbers for the regular theme?

Reply:
Good catch. Teresa confirmed that those numbers are in fact legacy and we can ignore them. I'll be sure to ask questions like that up front from now on.

Reply:
Awesome, thanks for confirming